AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift

```diff
@@ -177,6 +189,45 @@ public class ConstructorIO: CIOSessionManagerDelegate {
         executeTracking(request, completionHandler: completionHandler)
     }
 
+    /// Track recommendation view
+    ///
+    /// - Parameters:
+    ///   - podID: Pod ID
+    ///   - url: The current page URL the user viewed the recommendations on
+    ///   - numResultsViewed: The count of results that is visible to the user
+    ///   - resultPage: The current page that recommedantion result is on
+    ///   - resultCount: The total number of recommendation results
+    ///   - sectionName The name of the autocomplete section the term came from
+    ///   - resultID: Identifier of result set
+    ///   - completionHandler: The callback to execute on completion.
+    public func trackRecommendationResultsView(podID: String, url: String, numResultsViewed: Int? = nil, resultPage: Int? = nil, resultCount: Int? = nil, resultPositionOnPage: Int? = nil, sectionName: String? = nil, resultID: String? = nil, completionHandler: TrackingCompletionHandler? = nil) {
```

Comment:
for `trackBrowseResultsLoaded` we do not surface the url as an argument, maybe we should do that as well and make the default "Not Available"

Reply:
Updated! 👍